src/dataflow/src/server.rs

```diff
@@ -50,6 +50,10 @@ use crate::source::timestamp::TimestampBindingRc;
 
 mod metrics;
 
+/// How frequently each dataflow worker sends timestamp binding updates
+/// back to the coordinator.
+static TS_BINDING_FEEDBACK_INTERVAL_MS: u128 = 1_000;
```

Comment:
is there a benefit to making this a const instead? i guess my question is: why static instead of const?